large_objects_test.go

```diff
@@ -7,23 +7,24 @@ import (
 	"testing"
 	"time"
 
-	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgconn"
-	"github.com/jackc/pgx/v5/pgxtest"
 )
 
 func TestLargeObjects(t *testing.T) {
 	t.Parallel()
 
+	// We use a very short limit to test chunking logic.
+	maxLargeObjectMessageLength = 2
+
 	ctx, cancel := context.WithTimeout(context.Background(), 120*time.Second)
 	defer cancel()
 
-	conn, err := pgx.Connect(ctx, os.Getenv("PGX_TEST_DATABASE"))
+	conn, err := Connect(ctx, os.Getenv("PGX_TEST_DATABASE"))
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	pgxtest.SkipCockroachDB(t, conn, "Server does support large objects")
+	// pgxtest.SkipCockroachDB(t, conn, "Server does support large objects")
```

Comment:
Importing `pgxtest` creates an import loop. Should I move this function to some other package? Like `pgxtest_cockroach`?

Reply:
The solution is to make a new file such as large_objects_private_test.go that is in the pgx package. This file should contain a public function that changes maxLargeObjectMethodLength.